dashboard/app/models/unit_group.rb

```diff
@@ -98,15 +100,24 @@ def self.seed_from_hash(hash)
     course_version = unit_group.course_version
 
     if course_version
-      resources_imported = (hash['resources'] || []).map do |resource_data|
+      teacher_resources_imported = (hash['resources'] || []).map do |resource_data|
         resource_attrs = resource_data.except('seeding_key')
         resource_attrs['course_version_id'] = course_version.id
         resource = Resource.find_or_initialize_by(key: resource_attrs['key'], course_version_id: course_version.id)
         resource.assign_attributes(resource_attrs)
         resource.save! if resource.changed?
         resource
       end
-      unit_group.resources = resources_imported
+      unit_group.resources = teacher_resources_imported
+      student_resources_imported = (hash['student_resources'] || []).map do |resource_data|
+        resource_attrs = resource_data.except('seeding_key')
```

Comment:
Maybe make the code inside the map a function since it looks like its the same for teacher and student resources?